src/components/DataView/DataViewColumnDrawerContent.tsx

```diff
@@ -1,165 +1,186 @@
 import React, { useState } from "react";
 import styled from "styled-components";
-import ArrowDownwardIcon from "@mui/icons-material/ArrowDownward";
-import ArrowUpwardIcon from "@mui/icons-material/ArrowUpward";
-
-import Button from "../Button";
+import DragIndicatorIcon from "@mui/icons-material/DragIndicator";
 import theme from "@root/theme";
 import CheckboxList from "@root/components/CheckboxList";
-import DrawerContent from "@root/components/DrawerContent";
 import { useMosaicTranslation } from "@root/i18n";
+import DrawerHeader from "../DrawerHeader";
+import {
+	DragDropContext,
+	Draggable,
+	Droppable,
+	DropResult,
+} from "react-beautiful-dnd";
+import { DataViewProps } from "./DataViewTypes";
+import { ButtonProps } from "../Button";
+
+const ColumnTitle = styled.h2`
+  color: ${theme.newColors.almostBlack["100"]};
+  font-size: 20px;
+  font-weight: ${theme.fontWeight.normal};
+  line-height: 24px;
+  margin: 0 0 24px 0;
+`;
 
 const StyledWrapper = styled.div`
+  display: flex;
+  margin-top: 30px;
+
+  & > .left {
+    border-right: 2px solid ${theme.newColors.grey2["100"]};
+    flex: 1;
+    padding: 0 32px 0 40px;
+    width: 265px;
+  }
+
+  & > .left .listItem {
+    margin: 3px 0px;
+    background: white;
+  }
+
+  & > .right {
+    flex: 1;
+		margin: 0 44px 0 32px;
+    width: 280px;
+  }
+`;
+
+const ColumItem = styled.div`

```

Comment:
There's a typo on the name
```suggestion
const ColumnItem = styled.div`

```